snap/validate.go

```diff
@@ -27,6 +27,7 @@ import (
 // Regular expression describing correct identifiers.
 var validName = regexp.MustCompile("^[a-z](?:-?[a-z0-9])*$")
 var validEpoch = regexp.MustCompile("^(?:0|[1-9][0-9]*[*]?)$")
+var validHookName = regexp.MustCompile(`^[a-z-]+$`)
```

Comment:
This should be closer to `validName` above, as it prevents double dashes and dashes starting and ending the name.